generator/content.py

```diff
@@ -151,7 +151,13 @@ def generate_folders(namespaces: dict):
     type=click.File(),
     help="Path to a yaml namespaces file",
 )
-def generate_content(namespaces):
+@click.option(
+    "--allowlist",
+    default="namespace_allowlist.yaml",
```

Comment:
I think this should be named `namespaces-allowlist.yaml`, because `namespaces` is consistently plural elsewhere, and because `custom-namespaces.yaml` is already using kebab-case instead of snake_case for file names.
```suggestion
    default="namespaces-allowlist.yaml",
```

Reply:
I agree - I'll update the filename.